src/containers/ListingPage/components/ToggleFilterChoices.jsx

```diff
@@ -0,0 +1,66 @@
+/*
+ *  Copyright (c) 2017-present, NDLA.
+ *
+ *  This source code is licensed under the GPLv3 license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import {
+    addEventListenerForResize,
+    updateIFrameDimensions,
+    addAsideClickListener,
+    removeEventListenerForResize,
+    removeAsideClickListener,
+} from 'ndla-article-scripts';
+
+import React, { PropTypes, Component } from 'react';
+import { ListingShape } from '../../../shapes';
+import FilterChoices from './FilterChoices';
+
+
+class ToggleFilterChoices extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isVisFilter: false,
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  componentDidMount() {
+    addEventListenerForResize();
```

Comment:
Slik jeg forsto @oyvinmar så trengtes disse bare på rot nivå for en listing. Stemmer det?